pindel/workflow.cwl

```diff
@@ -90,17 +94,6 @@ steps:
             reference: reference
             cram: tumor_cram
             vcf: region_filter/filtered_vcf
+            output_vcf_basename: "pindel"
```

Comment:
It seems to still try to interpret "pindel" instead of taking it as a string literal. I wonder if you need to use something like   valueFrom?